frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx

```diff
@@ -397,8 +397,8 @@ export function LineGraph_({
             }
         }
 
-        const annotations = alertLines.reduce((acc, { value }, idx) => {
-            acc[`${idx}`] = {
```

Comment:
Everything looks good to me, but maybe @anirudhpillai knows if there's a reason to stringify these keys

Reply:
I think it was made to cast a type, but I've fixed the type cast in the line below.